ydb_sqlalchemy/sqlalchemy/__init__.py

```diff
@@ -853,25 +832,43 @@ def _add_declare_for_yql_stmt_vars_impl(self, statement, parameters_types):
         )
         return f"{declarations}\n{statement}"
 
+    def __merge_parameters_values_and_types(
+        self, values: Mapping[str, Any], types: Mapping[str, Any], execute_many: bool
+    ) -> Sequence[Mapping[str, ydb.TypedValue]]:
+        if isinstance(values, collections.abc.Mapping):
+            values = [values]
+
+        result_list = []
+        for value_map in values:
+            result = {}
+            for key in value_map.keys():
+                if key in types:
+                    result[key] = ydb.TypedValue(value_map[key], types[key])
+                else:
+                    result[key] = value_map[key]
+            result_list.append(result)
+        return result_list if execute_many else result_list[0]
+
     def _make_ydb_operation(
```

Comment:
**nitpick (non-blocking)**: It doesn't make `ydb_operation` anymore, so let's rename it